src/explore-education-statistics-admin/src/pages/release/summary/ReleaseSummaryForm.tsx

```diff
@@ -87,11 +87,38 @@ const ReleaseSummaryForm = <FormValues extends EditFormValues>({
     return optGroups;
   };
 
+  const months = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+  const todaysDate = new Date();
+
   const baseValidationRules: ObjectSchemaDefinition<EditFormValues> = {
     timePeriodCoverageCode: Yup.string().required('Choose a time period'),
     timePeriodCoverageStartYear: Yup.string().required('Enter a year'),
     releaseTypeId: Yup.string().required('Choose a release type'),
-    scheduledPublishDate: validateMandatoryDayMonthYearField,
+    scheduledPublishDate: validateMandatoryDayMonthYearField.test(
+      'validDateIfAfterToday',
+      `Schedule publish date can't be before ${todaysDate.getDate()} ${
+        months[todaysDate.getMonth()]
+      } ${todaysDate.getFullYear()}`,
+      value => {
+        const date = new Date(value.year, value.month, value.day);
```

Comment:
I know this has been copy and pasted from other areas, but I don't think this is a great way of actually constructing the date as it could potentially introduce weird bugs as Javascript's Date constructor is weird (and is just as flawed as Java's)

This one's probably an interesting one for you to know about too @duncan-at-hiveit 

For examples of how hilariously broken Date actually is:
- A year of 99 is considered to mean 1999. 
- Putting in numbers larger than 12 for the month will rollover, so 14 means March.
- Rollover also happens for the day, so 32 can mean the 1st.

I would suggest that we should be using date-fn's `parse` method using a date format of `yyyy-M-d`. Unlike the Date constructor, this will return an invalid Date object that we can actually validate against with date-fns `isValid` function.

I think this should also be back-applied to all other instances where we're using `new Date(value.year, value.month, value.day)`.

Reply:
This should be back-applied to other validators.

Reply:
This still isn't resolved

Reply:
Interesting, thanks @ntsim @HiveIT-Lewis it's good to know